app/src/app/providers.tsx

```diff
@@ -46,13 +46,26 @@ const poppins = localFont({
 
 export function Providers({ children }: { children: React.ReactNode }) {
   return (
-    <>
+    <div className={`${openSans.className} ${poppins.className}`}>
+      <link rel="preconnect" href="https://fonts.googleapis.com" />
+      <link
+        rel="preconnect"
+        href="https://fonts.gstatic.com"
+        crossOrigin="anonymous"
+      />
+      <link
+        href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&family=Poppins:wght@400;500;600&display=swap"
+        rel="stylesheet"
+      />
```

Comment:
Let's host the fallback font locally or specify better default fallback fonts that are available in the operating system (e.g. at least something that's sans-serif).